collections/_infographics/energy/electricity-czechia.md

```diff
@@ -3,15 +3,30 @@ layout:        infographic
+
+* End-user electricity consumption, which experienced a slight decline after 2008, has been growing steadily over the last five years<!-- (see page 12, [ERO: Annual Report on the Operation of the EC of the Czech Republic 2018]http://www.eru.cz/documents/10540/4580207/Rocni_zprava_provoz_ES_2018.pdf/))-->. __Last year, end-user consumption recorded its highest values in the whole period under review since 1983__. The largest increase was recorded at the high and very high voltage levels, i.e. for the more energy-intensive operations. On the other hand, consumption decreased for households and small businesses<!--, see the analysis of [TZB-info](https://energetika.tzb-info.cz/elektroenergetika/19020-spotreba-elektriny-byla-v-roce-2018-nejvyssi-za-cele-sledovane-obdobi)-->. The industrial sector accounts for the largest share of consumption (30.7%), followed by households (24.7%) and the trade, services, education and health sector (20.5%).
+
+## Methodology used and origin of data
+
+* To calculate emissions, we used data on gross electricity production by individual types of power plants or fuels used from the annual report on the operation of the Czech electricity system in 2018<!--[ERO: Annual Report on the Operation of the EC of the Czech Republic 2018](http://www.eru.cz/documents/10540/4580207/Rocni_zprava_provoz_ES_2018.pdf/)-->. Based on these data, we calculated the corresponding emissions using the relevant IPCC coefficients ([IPCC 2014: ANNEX III, Technology-specific Cost and Performance Parameters](https://www.ipcc.ch/site/assets/uploads/2018/02/ipcc_wg3_ar5_annex-iii.pdf)). for its uniformity and data availability from different sources. For example, the authors of the [ElectricityMap](https://www.electricitymap.org/map?lang=en) project used a similar approach.
```

Comment:
Same applies here. Unable to find the English reference for http://www.eru.cz/documents/10540/4580207/Rocni_zprava_provoz_ES_2018.pdf.